### ISSUE TITLE
[SelectMenuOption]: AccessibleName of SelectMenuOption not announced by VoiceOver and JAWS.

### ISSUE THREAD

**Issue by witja46** [SAP/ui5-webcomponents#11809]
### Describe the bug

Screen readers (VoiceOver on Mac and JAWS on Windows) do not announce the accessibleName text of the SelectMenuOption in both Chrome and Firefox.

### Isolated Example

https://stackblitz.com/edit/github-d4xfy9u1?file=src%2FApp.tsx

### Reproduction steps

Using VoiceOver in Chrome:
1. Tab to the  Select component
2. Open it by pressing enter
3. Listen to the Screen Reader announcement of the first SelectMenuOption
4. Voice Over announces: "A..z list position 1 of 4"
...

### Expected Behaviour

Text specified with the accessibleName("Sort in alphabetical ascending order") is read to the user followed by the position of the option inside of the options list.

### UI5 Web Components for React Version

1.27.0

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

MacOS and Windows 11

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-p, 
The accessibleName of the Select dos not seem to work properly. 
Please have a look. 

Kind regards, 
Georgi Valkanov
SAP Development Support

**Comment by NakataCode** [comment]
Hello @witja46! 

Thank you very much for your feedback and incident report.
Unfortunately, it turned out that the components you are using (`SelectMenu` and `SelectMenuOption`) are no longer supported. These components were [officially removed](https://sap.github.io/ui5-webcomponents/nightly/docs/migration-guides/to-version-2/#ui5-select-menu-ui5-select-menu-option) in UI5 Web Components v2.0 as part of a migration to a cleaner API design.

**Migration path:**
Please refer to our official migration guide: https://sap.github.io/ui5-webcomponents/docs/migration-guides/to-version-2/ 
This shows how to update from the old pattern to the new one that supports proper accessibility. 

I'm going to close this issue since it's related to using removed components, but feel free to reach out if you have questions.

Best Regards,
Nikola

**Comment by NakataCode** [comment]
Hello,

The [fix](https://github.com/UI5/webcomponents/pull/13202) has been merged. I will proceed with closing this issue.

Best regards,
Nikola
